cli/module_registry.go

```diff
@@ -176,12 +176,12 @@ func UpdateModuleAction(c *cli.Context) error {
 
 // UploadModuleAction is the corresponding action for 'module upload'.
 func UploadModuleAction(c *cli.Context) error {
-	manifestPathArg := c.String("module")
-	publicNamespaceArg := c.String("public-namespace")
-	orgIDArg := c.String("org-id")
-	nameArg := c.String("name")
-	versionArg := c.String("version")
-	platformArg := c.String("platform")
+	manifestPathArg := c.String(moduleFlagPath)
+	publicNamespaceArg := c.String(moduleFlagPublicNamespace)
+	orgIDArg := c.String(moduleFlagOrgID)
+	nameArg := c.String(moduleFlagName)
+	versionArg := c.String(moduleFlagVersion)
+	platformArg := c.String(moduleFlagVersion)
```

Comment:
```suggestion
	platformArg := c.String(moduleFlagPlatform)
```

Reply:
😮‍💨 thanks for catching. Looked through one more time to double check I didn't mess up any others...